packages/v1-sdk/src/common/api/amm/getAmmInfo.ts

```diff
@@ -1,28 +1,22 @@
-import { getServiceUrl } from '../urls';
-import { RawAMM, AMMInfo } from './types';
+import { V1V2Pool } from '@voltz-protocol/api-v2';
```

Comment:
this is a bit of a confusing name:)

Reply:
We can rename it at some point but it's out-of-scope of this PR.